R/reactData.R

```diff
@@ -10,6 +10,15 @@ asReactData <- function(x) UseMethod("asReactData")
 #' @export
 asReactData.ReactData <- function(x) x
 
+#' @export
+asReactData.html <- function(x) {
+  ReactData(
+    type = "html",
+    value = dropDeps(x),
+    deps = getDeps(x)
+  )
+}
```

Comment:
Thanks, I overlooked this. This PR has been opened since March and only now I came back to revisit it 😅 Indeed adding this function to namespace makes `isHTML` obsolete

Reply:
The behavior with app you provided is the expected behavior. I added some screenshot tests to capture this behavior. I hope that since those are very small test apps those screenshots will be quite resilient to changes. I tried testing just HTML with `app$get_html()` but the case when HTML is not rendered as HTML, but as a string produces very obscure strings `"<div class=\"react-container\">&lt;span style='font-weight: bold;'&gt;Hello &lt;span style='font-weight: normal;'&gt;from ReactContext in UI without htmltools::HTML&lt;/span&gt;&lt;/span&gt;</div>"` and I think it will be easier to manage screenshots than such strings as test expecations.